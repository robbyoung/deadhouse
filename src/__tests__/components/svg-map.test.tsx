import React from 'react';
import renderer from 'react-test-renderer';
import SvgMap from '../../components/svg-map';

// react-svg-pan-zoom throws errors while trying to work with the test DOM
jest.mock('react-svg-pan-zoom');

describe('SvgMap component', () => {
    it('Renders an SVG image', () => {
        const component = renderer.create(<SvgMap src="genabackis-map.svg" />);

        expect(component).toMatchSnapshot();
    });
});
