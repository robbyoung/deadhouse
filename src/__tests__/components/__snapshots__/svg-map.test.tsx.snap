// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgMap component Renders an SVG image 1`] = `
<div>
  <div />
</div>
`;
